src/lib/constants.ts

```diff
@@ -21,18 +21,49 @@ export const fallback = {
 
 export const MAINNET_CHAIN_ID = 1;
 export const GOERLI_CHAIN_ID = 5;
+export const SEPOLIA_CHAIN_ID = 11155111;
 
-type SupportedNetworks = "mainnet" | "goerli";
+type SupportedNetworks = "mainnet" | "goerli" | "sepolia";
 export const supportedNetworks: {
   [key: number]: SupportedNetworks;
 } = {
   [MAINNET_CHAIN_ID]: "mainnet",
   [GOERLI_CHAIN_ID]: "goerli",
+  [SEPOLIA_CHAIN_ID]: "sepolia",
 };
 
 export const chainIdBlockOffsets: {
   [key: number]: number;
 } = {
   [MAINNET_CHAIN_ID]: 0,
   [GOERLI_CHAIN_ID]: 24,
+  [SEPOLIA_CHAIN_ID]: 24,
 };
+
+export const flashbotsSupportedNetworks: {
+  [key in SupportedNetworks]: {
+    name: string;
+    chainId: number;
+    streamUrl: string;
+    apiUrl: string;
+  }
+} = {
+  mainnet: {
+    name: "mainnet",
+    chainId: 1,
+    streamUrl: "https://mev-share.flashbots.net",
+    apiUrl: "https://relay.flashbots.net"
+  },
+  goerli: {
+    name: "goerli",
+    chainId: 5,
+    streamUrl: "https://mev-share-goerli.flashbots.net",
+    apiUrl: "https://relay-goerli.flashbots.net"
+  },
+  sepolia: {
```

Comment:
MEV Share lib @flashbots/mev-share-client [doesn't include sepolia](https://github.com/flashbots/mev-share-client-ts/blob/5b937dd4fc21c75aad26a8fb969c7e40ff73cacd/src/api/networks.ts#L7)

But it's supported: https://docs.flashbots.net/flashbots-auction/advanced/rpc-endpoint